src/test/java/com/amazon/ion/TextToBinaryUtils.kt

```diff
@@ -64,4 +81,29 @@ object TextToBinaryUtils {
     fun ByteArray.byteArrayToBitString(): String {
         return this.joinToString(" ") { it.toUByte().toString(2).padStart(8, '0') }
     }
+
+    /**
+     * Converts a byte array to a string of hex bytes, such as "A5 0F EC 52".
+     * The purpose of this method is to make it easier to read and write test assertions.
+     */
+    @JvmStatic
+    fun ByteArray.byteArrayToHexString(): String {
+        return this.joinToString(" ") { it.toUByte().toString(16).padStart(2, '0') }
+    }
+
+    /**
+     * Converts a string of decimal integers, such as "105 -9349549 0 -12 99999", to an int array.
+     */
+    @JvmStatic
+    fun String.decimalStringToIntArray(): IntArray {
+        return octetStringToIntArray(this, 10)
+    }
```

Comment:
I think this needs more explanation. Since the decimal octets aren't a fixed size, what do we do if a number is too large to fit in an integer?

Why is this necessary when you can just do `intArrayOf(105, -9349549, 0, -12, 99999)`?